### PR TITLE
(0.31.0) Fix JIT code breakpoints on x86

### PR DESCRIPTION
**Pull request by 0xdaryl**
Code breakpoints inserted by the JIT should use an `int3` instruction rather
than the `bad` pseudo instruction.

master branch PR: #14435

Signed-off-by: Daryl Maier <maier@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
Pls refer to the original PR in the description.

**Comment by 0xdaryl** [comment]
This should be a fairly safe fix for JDK18.  JITed method breakpoints are broken without it.  This problem was introduced by some refactoring work done last year.

@dsouzai @pshipton

**Review by dsouzai — APPROVED** [review]

**Comment by dsouzai** [comment]
@pshipton I'm good to merge if everything looks good to you.

**Comment by pshipton** [comment]
The change is fine with me, although I'd wait until tomorrow to merge once the original PR has gone through the nightly builds.

**Review by pshipton — APPROVED** [review]

**Pull request merged**
